streams/actions/src/main/java/com/hortonworks/streamline/streams/actions/topology/service/TopologyTestRunner.java

```diff
@@ -181,6 +184,15 @@ public TopologyTestRunHistory runTest(TopologyActions topologyActions, Topology
         return history;
     }
 
+    @edu.umd.cs.findbugs.annotations.SuppressWarnings(
+            value="RV_RETURN_VALUE_IGNORED_BAD_PRACTICE",
+            justification = "We don't have to deal with the return values from mkdirs() and createNewFile()")
```

Comment:
mkdirs returns false if "the dir already exists" or "dir does not exist but could not be created". Maybe you can check if dir exists before invoking mkdirs and then check for its return code ?

Reply:
Yes that would be better. I'll address it.